src/components/Artists/Artists.js

```diff
@@ -81,13 +21,25 @@ export class ArtistsComponent {
     });
   }
 
-  mount() {
+  fetchArtistsData() {
+    this.state.isFetching = true;
```

Comment:
please remove isFetching for now. We will deal with loader later

Reply:
Fix